### ISSUE TITLE
Unblocking is not explained in the specification

### ISSUE THREAD

**Issue by capjamesg** [indieweb/microsub#51]
The specification does not currently include a section on the "unblock" action which is mentioned in two other places in the specification, including the "Actions" list at https://indieweb.org/Microsub-spec#Actions.

While one could assume what an unblocking request should look like given the structure of the mute and unmute functions which are relatively similar, the spec should clarify exactly how one should go about issuing an Unblock request.

**Comment by jalcine** [comment]
i agree. I think mirroring the output of mutes and unmutes will suffice
